services/address-lookup/README.md

```diff
@@ -0,0 +1,97 @@
+# address-lookup
+
+A service to retrieve UK postcodes by the postcode.
+
+The service consumes the OS Apis (https://apidocs.os.uk/), specifically the DPA dataset (https://apidocs.os.uk/docs/os-places-dpa-output).
+
+## Installation
+
+    npm i @planning-inspectorate/address-lookup
+
+And add the API key in  .env of your project:
+
+    OS_API_KEY = {insert key}
+
+You can then use the service importing
+
+    import {findAddressListByPostcode} from "@planning-inspectorate/address-lookup";
+    
+    const {errors, addressList} = await findAddressListByPostcode('EC2M7PD');
+
+
+## Params
+The function ``findAddressListByPostcode`` needs a ``string`` parameter: the postcode.
+
+The postcode does not get validated inside the function: in case it's not valid the API will just return an empty list of addresses.
```

Comment:
Does the string need to have a space? Can we pass partial strings?

Reply:
If you haven't already, could we just add what you have said here to the README, please.